cmd/swagger/commands/diff/reporting.go

```diff
@@ -81,15 +81,15 @@ func primitiveTypeString(typeName, typeFormat string) string {
 	return typeName
 }
 
-// TypeDiff - describes a primitive type change
+// TypeDiff - describes a primitive type change.
 type TypeDiff struct {
-	Change      SpecChangeCode `json:"change-type,omitempty"`
+	Change      SpecChangeCode `json:"change_type,omitempty"`
 	Description string         `json:"description,omitempty"`
-	FromType    string         `json:"from-type,omitempty"`
+	FromType    string         `json:"from_type,omitempty"`
 	ToType      string         `json:"to-type,omitempty"`
```

Comment:
Interesting; so it did complain about `from-type` -> `from_type`, but did not for `to-type` (should this also be `to_type` ?